mlflow/pytorch/__init__.py

```diff
@@ -593,6 +681,33 @@ def _load_model(path, **kwargs):
             return torch.jit.load(model_path)
 
 
+@experimental
+def load_state_dict(model_uri):
+    """
+    Load a state dict from a local file or a run.
+
+    :param model_uri: The location, in URI format, of the MLflow model, for example:
+
+                    - ``/Users/me/path/to/local/model``
+                    - ``relative/path/to/local/model``
+                    - ``s3://my_bucket/path/to/model``
+                    - ``runs:/<mlflow_run_id>/run-relative/path/to/model``
+                    - ``models:/<model_name>/<model_version>``
+                    - ``models:/<model_name>/<stage>``
```

Comment:
Can we fix the parameter description here because `load_state_dict` doesn't load an MLflow model?

Reply:
removed `models:/`. Since we doenload the model using `_download_artifacts_from_uri`, the first 4 elements should still work. Correct me if am wrong.